test/integration/patients/patient/sidebar.js

```diff
@@ -451,6 +453,23 @@ context('patient sidebar', function() {
               },
             },
           }),
+          addWidget({
+            id: 'patientMRNIdentifier',
+            widget_type: 'patientIdentifiers',
+            definition: {
+              display_name: 'Patient Identifier',
+              identifier_type: 'mrn',
+            },
+          }),
+          addWidget({
+            id: 'patientSSNIdentifier',
+            widget_type: 'patientIdentifiers',
+            definition: {
+              default_html: 'No Identifier Found',
+              display_name: 'Patient Identifier With Empty Value',
+              identifier_type: 'ssn',
+            },
+          }),
```

Comment:
`patientMRNIdentifier` widget is for testing that an identifier value is displayed properly.

`patientSSNIdentifier` widget is for testing that `default_html` is used when the identifier value is empty or `null`.